src/Wallabag/CoreBundle/Resources/translations/messages.en.yml

```diff
@@ -46,13 +46,13 @@ menu:
         input_label: 'Enter your search here'
 
 footer:
-    wallabag:
```

Comment:
Can't we configure weblate to use 4 spaces indentation instead of 2?

Reply:
Yes, we can. Do you mind doing this afterwards though, so that I don't need to rebase everything? :sob: 
![image](https://user-images.githubusercontent.com/2197836/80349772-07651080-8870-11ea-9e63-75e5ec202d57.png)

Reply:
@tcitworld you should play with `git rerere` :sweat_smile:

Reply:
I can't find that _parameter_ page.

Reply:
Project > Composant > Greffons > Personnaliser la sortie YAML > Installer.

Reply:
Just did it.